app/Services/PlanetService.php

```diff
@@ -473,7 +467,7 @@ public function getBuildingConstructionTime(string $machine_name): int
             (
                 ($price->metal->get() + $price->crystal->get())
                 /
-                (2500 * max((4 - ($next_level / 2)), 1) * (1 + $robotfactory_level) * $universe_speed * pow(2, $nanitefactory_level))
+                (2500 * max((4 - ($next_level / 2)), 1) * (1 + $robotfactory_level) * $universe_speed * ( 2 ** $nanitefactory_level ) )
```

Comment:
Please remove spaces surrounding () to adhere to PSR-2 standards.

Btw, 👍  I didn't know that the `**` operator exists. Nice!